invoice.go

```diff
@@ -217,6 +219,19 @@ type InvoiceCustomField struct {
 	Value string `json:"value"`
 }
 
+// InvoiceThresholdReason is a structure representing a reason for a billing threshold.
+type InvoiceThresholdReason struct {
+	AmountGte   int64                               `json:"amount_gte"`
```

Comment:
I'm assuming gte = "greater than or equal". Can we upcase it in all instances to be `GTE`? Things like `ID`, `PDF`, or `URL` are pretty ubiquitous convention through the library.